tests/webapp/api/test_performance_data_api.py

```diff
@@ -393,6 +393,7 @@ def test_perf_summary(client, test_perf_signature, test_perf_data):
         'test': test_perf_signature.test,
         'lower_is_better': test_perf_signature.lower_is_better,
         'has_subtests': test_perf_signature.has_subtests,
+        'measurement_unit': None,
```

Comment:
I think we can provide a `measurement_unit` in the `tests/conftest.py::test_perf_signature fixture.
It would better simulate a real-world example.

Reply:
Yep.